src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2701,8 +2701,10 @@ def deallocate_vmss(cmd, resource_group_name, vm_scale_set_name, instance_ids=No
         return sdk_no_wait(no_wait, client.virtual_machine_scale_set_vms.begin_deallocate,
                            resource_group_name, vm_scale_set_name, instance_ids[0])
 
+    VirtualMachineScaleSetVMInstanceIDs = cmd.get_models('VirtualMachineScaleSetVMInstanceIDs')
+    vm_instance_i_ds = VirtualMachineScaleSetVMInstanceIDs(instance_ids=instance_ids)
```

Comment:
Can we use `vm_instance_ids`? Why split `ids`?

Reply:
That can be changed, but I split ids the way I did because it matches the SDK's parameter name [here](https://docs.microsoft.com/en-us/python/api/azure-mgmt-compute/azure.mgmt.compute.v2020_12_01.operations.virtualmachinescalesetsoperations?view=azure-python#begin-deallocate-resource-group-name--vm-scale-set-name--vm-instance-i-ds-none----kwargs-).